src/main/java/io/dashboardhub/pipelinedashboard/domain/Project.java

```diff
@@ -8,17 +11,15 @@
 import java.util.UUID;
 
 @Data
+@EqualsAndHashCode(callSuper = true)
+@ToString(callSuper = true)
 @Entity
+@EntityListeners(AuditingEntityListener.class)
 @Table(uniqueConstraints = @UniqueConstraint(name = "uid_idx", columnNames = "uid"))
-public final class Project {
-
-    @Id
-    @GeneratedValue(strategy = GenerationType.AUTO)
-    private Long id;
+public class Project extends BaseEntity {
```

Comment:
merge issue. Still not sure why we want to mark all the entities as final though ...